python/tests/test_account.py

```diff
@@ -1705,9 +1700,7 @@ def test_synchronize_member_list_on_group_rejoin(self, acfactory, lp):
 
         assert len(msg.chat.get_contacts()) == len(chat.get_contacts())
 
-        ac1.shutdown()
-        ac2.shutdown()
-        ac3.shutdown()
+        acfactory.dump_imap_structures(sys.stdout)
```

Comment:
Are you sure you wanted to commit this line?

Reply:
yes, i want to test here that the helper function works and get it as a regular failing test if not.